packages/core/package.json

```diff
@@ -63,8 +63,8 @@
   },
   "dependencies": {
     "@cardano-ogmios/client": "5.5.7",
-    "@cardano-sdk/util": "^0.7.0",
     "@cardano-ogmios/schema": "5.5.7",
+    "@cardano-sdk/util": "^0.7.0",
```

Comment:
Not sure why this is needed, `yarn` insisted on it

Reply:
Alphabetical reordering by the looks of things

Reply:
Yeah seems that way, but why wasn't it ordered when originally added? :thinking:

Reply:
Maybe it was added manually with an older version of `yarn` and there's since been a change in a later version.